libecho/src/BitReceiver.cc

```diff
@@ -228,3 +228,12 @@ int BitReceiver::receive(uint8_t *buffer, int size) {
 
     return size;
 }
+
+void BitReceiver::wait() {
```

Comment:
Maybe rename it to something like skip? I think that wait doesn't explain enough well what this method does.